arduino/Sender/Sender.ino

```diff
@@ -36,12 +36,12 @@ unsigned int soilmoisture;
 void setup() {
   // Initialize devices
   serial_init();
-  led_init();
```

Comment:
why removed this?

Reply:
Because so far we have assumed that our pro-mini clone was an exact copy of the original pro-mini, but or board had the LED on D13(not D9) which is used by the SPI. So our led function did not do anything actually...